consensus/state.go

```diff
@@ -1518,6 +1515,7 @@ func (cs *State) enterPrecommitWait(height int64, round int32) {
 	defer func() {
 		// Done enterPrecommitWait:
 		cs.TriggeredTimeoutPrecommit = true
+		cs.updateRoundStep(round, cstypes.RoundStepPrecommitWait)
 		cs.newStep()
```

Comment:
We also have one called in[ newStep()](https://github.com/celestiaorg/celestia-core/blob/602544164e68fca964ae2dd16e09eadc7ba7c5ed/consensus/state.go#L711), wouldn't that cause duplicates?

Reply:
I misread the code, the change is fine!